opendkim/opendkim.8.in

```diff
@@ -282,6 +282,11 @@ Normally
 forks and exits immediately, leaving the service running in the background.
 This flag suppresses that behaviour so that it runs in the foreground.
 .TP
+.I \-g
+Set CheckSigningTable to no.  This means that when the config is loaded,
+The SigningTable will not be checked for any missing keys in
+the KeyTable.
```

Comment:
* A typo " the SiningTable"
* The mention to `CheckSigingTable` option is not so important for understanding what this commandline option does.  So it is good to move it after the last sentence of the paragraph, with reference information to opendkim.conf(5).

Reply:
I see a typo, but in your comment, not the code.
Anyway I re-wrote it based on your suggestion.
See commit https://github.com/trusteddomainproject/OpenDKIM/commit/3dabd5fc7f5151ad0eed067661aeb6c33b38d8ca